From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Kenneth Myhra <kennethmyhra@gmail.com>
Date: Thu, 12 Aug 2021 20:22:35 +0200
Subject: [PATCH] Use glib's in-built C_IN

Since we do not have C_IN and glib has functionality for providing it,
let glib provide it.
---
 gio/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gio/meson.build b/gio/meson.build
index 7adfd0ea8a9845bab29c5c64aba288bd7d096d8b..db74f44e19d8f19bfc6846c5cce2f6f278552add 100644
--- a/gio/meson.build
+++ b/gio/meson.build
@@ -18,7 +18,7 @@ gnetworking_h_conf = configuration_data()
 
 gnetworking_h_nameser_compat_include = ''
 
-if host_system not in ['windows', 'android']
+if host_system not in ['windows', 'android', 'serenity']
   # Don't check for C_IN on Android since it does not define it in public
   # headers, we define it ourselves wherever necessary
   if not cc.compiles('''#include <sys/types.h>
